src/main/java/com/alkemy/ong/controller/MemberController.java

```diff
@@ -0,0 +1,36 @@
+package com.alkemy.ong.controller;
+
+import com.alkemy.ong.dto.MemberDto;
+import com.alkemy.ong.service.EmailService;
+import com.alkemy.ong.service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.MessageSource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/members")
+public class MemberController {
+    private final MessageSource messageSource;
+
+    @Autowired
+    private MemberService memberService;
+
+    @PostMapping
+    public ResponseEntity<Map<String,String>> memberCreate(@Valid @RequestBody MemberDto memberDto) {
+        memberService.saveMember(memberDto);
+        Map<String, String> message = new HashMap<>(){{put("message: ", messageSource.getMessage("member.created.successfully",new Object[]{memberDto.getName()}, Locale.ENGLISH));}};
```

Comment:
Me parecen muy piolas los mensajes de informacion cuando es satisfactorio, habria que implementarlo en todos los casos, le agrega bastante valor a una demostracion de API Rest, ya que no tenemos UI

Reply:
Gracias profesor, dale. Le voy a decir a los chicos para agregarlos a todos los metodos create. En cuanto a lo primero que dijo, le paso dos parametros solamente porque la entidad solo tiene 2 parametros not null.